src/com/createsend/Journeys.java

```diff
@@ -0,0 +1,19 @@
+package com.createsend;
+
+import com.createsend.models.journeys.JourneySummary;
+import com.createsend.util.AuthenticationDetails;
+import com.createsend.util.JerseyClientImpl;
+import com.createsend.util.exceptions.CreateSendException;
+
+public class Journeys extends CreateSendBase {

```

Comment:
Nitpick: Shouldn't this be named `Journey`, rather than `Journeys`; it takes a journey id after all.

Reply:
I'll leave it as is to be consistent with the other endpoint classes. i.e:
- https://github.com/campaignmonitor/createsend-java/blob/master/src/com/createsend/Lists.java
- https://github.com/campaignmonitor/createsend-java/blob/master/src/com/createsend/Campaigns.java